client/cmd/bisonw-desktop/pkg/pkg-darwin.sh

```diff
@@ -94,14 +96,16 @@ function build_targets() {
 
     echo "Building .DMG click installer for ${OS}-${ARCH}"
 
-	TARGET_NAME="bisonw${FLAVOR}-${OS}-${ARCH}-v${VER}"
+	TARGET_NAME="bisonw-desktop-${OS}-${ARCH}-v${VER}"
 
 	# Remove any existing executable if any.
 	rm -rf "${APP_EXCE_DIR}"
 	mkdir -p "${APP_EXCE_DIR}"
 
+	# Potentially want to run with CGO_CXXFLAGS="-mmacosx-version-min=10.11.0"
+
```

Comment:
Can add it now I guess.

Reply:
It's not clear yet that this is necessary. Have you investigated?

Reply:
It's not necessary. It's just an extra constraint.

Reply:
We add it when we need it.